hw26/index.js

```diff
@@ -0,0 +1,13 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    function structureUserInfo(name) {
+        let result = {};
+        result.name = name;
+        return function(role) {
+            result.role = role;
+            return result;
+        }
```

Comment:
Pretty nice. However, we definitely could simplify it

```suggestion
        return role => ({ role, name })
```